plugins/inputs/t128_graphql/sample.go

```diff
@@ -13,17 +13,17 @@ var sampleConfig = `
 # unix_socket = "/var/run/128technology/web-server.sock"
 
 ## The starting point in the graphQL tree for all configured tags and fields
-# entry_point = "allRouters[name:RTR_WEST_COMBO]/nodes/nodes[name:combo-west]/nodes/arp/nodes"
+# entry_point = "allRouters[name:${ROUTER}]/nodes/nodes[name:${NODE}]/nodes/arp/nodes"
```

Comment:
I think the string replacement for these variables are done via the monitoring agent and telegraf doesn't have this concept. Since this is what telegraf uses for the sample, I'd say leave it to be what you had

Reply:
I agree!